e2e/testdata/fn-render/empty-pipeline/Kptfile

```diff
@@ -2,4 +2,3 @@ apiVersion: kpt.dev/v1alpha2
 kind: Kptfile
 metadata:
   name: app
-pipeline:
```

Comment:
fn render formats `Kptfile` and that's resulting in removing this empty pipeline causing this test to fail. I am still not sure why we are not seeing this issue on `main` currently.